config/manager/deployment.yaml

```diff
@@ -25,14 +25,21 @@ spec:
         ports:
           - containerPort: 8080
             name: http-prom
-        livenessProbe:
-          httpGet:
-            port: http-prom
-            path: /metrics
+          - containerPort: 9440
+            name: healthz
+            protocol: TCP
         args:
-          - --enable-leader-election
-          - --log-level=debug
+          - --log-level=info
```

Comment:
The required args are:
```
          - --watch-all-namespaces
          - --log-level=info
          - --log-json
          - --enable-leader-election
```

Reply:
Yup, I have that in my stash, will commit.